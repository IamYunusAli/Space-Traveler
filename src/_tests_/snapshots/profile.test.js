import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../../redux/configureStore';
import Profile from '../../components/profile/profile';

describe('Profile', () => {
  test('should match Profile snapshot', () => {
    const tree = render(
      <Provider store={store}>
        <Profile />
      </Provider>,
    );
    expect(tree).toMatchSnapshot();
  });
});
