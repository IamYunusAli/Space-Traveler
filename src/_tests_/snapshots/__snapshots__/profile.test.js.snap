// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile should match Profile snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="my-profile"
      >
        <div
          class="profiles"
        >
          <h2>
            Rockets
          </h2>
          <div
            class="reserved"
          />
        </div>
        <div
          class="profiles"
        >
          <h2>
            My Missions
          </h2>
          <h4>
            No Joined Missions
          </h4>
        </div>
        <div
          class="profiles"
        >
          <h2>
            Dragons
          </h2>
          <div
            class="reserved"
          >
            <h4>
               0 Reserved Dragons  
            </h4>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="my-profile"
    >
      <div
        class="profiles"
      >
        <h2>
          Rockets
        </h2>
        <div
          class="reserved"
        />
      </div>
      <div
        class="profiles"
      >
        <h2>
          My Missions
        </h2>
        <h4>
          No Joined Missions
        </h4>
      </div>
      <div
        class="profiles"
      >
        <h2>
          Dragons
        </h2>
        <div
          class="reserved"
        >
          <h4>
             0 Reserved Dragons  
          </h4>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
